spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -145,14 +146,29 @@ export function connectToSpotTV(joinCode, shareMode) {
          * @returns {Promise}
          */
         function onDisconnect(error) {
-            logger.error('On Spot Remote disconnect', { error });
+            const usingPermanentPairingCode = getPermanentPairingCode(getState()) === joinCode;
+            const isUnrecoverableError = remoteControlClient.isUnrecoverableRequestError(error);
```

Comment:
Maybe rename to isRecoverableError? Maybe that should have been the method name this whole time and it never occurred to me.

Reply:
yeah, but can it be a separate PR?